docs/CHANGELOG.asciidoc

```diff
@@ -39,6 +40,309 @@ This section summarizes the changes in each release.
 * <<release-notes-7.0.0-alpha1>>
 
 --
+[[release-notes-7.9.0]]
+== {kib} 7.9.0
+coming[7.9.0]
+
+//See also <<breaking-changes-7.9,breaking changes in 7.9>>.
+
+Operations::
+* Moves kibana-keystore from data/ to config/ {kibana-pull}57856[#57856]
+
+[float]
+[[enhancement-7.9.0]]
+=== Enhancements
+APM::
+* Shows `trace.id`, `transaction.id` and/or `error.id` in metadata table {kibana-pull}66376[#66376]
+* Adds error rate chart to Transaction overview and detail views {kibana-pull}67327[#67327]
+* Adds ThemeProvider to support dark mode {kibana-pull}68242[#68242]
+* Triggers Lazy-load alert {kibana-pull}68806[#68806]
+* Changes to duration formatting {kibana-pull}69039[#69039]
+* Adds callout to inform users of high cardinality in unique transaction names {kibana-pull}69112[#69112]
+* Creates API to return data to be used on the Overview page {kibana-pull}69137[#69137]
+* Fixes confusing request/minute viz {kibana-pull}69143[#69143]
+* Adds decimals only for numbers below 10 {kibana-pull}69334[#69334]
+* Adds support for dark mode {kibana-pull}69362[#69362]
+* Chart breakdowns {kibana-pull}69420[#69420]
+* Quotes trace id to ensure a word is searched (#69500) {kibana-pull}69504[#69504]
+* Adds error rates to Service Map popovers {kibana-pull}69520[#69520]
+* Resubmits initial version {kibana-pull}69531[#69531]
+* Adds 'Anomaly detection' settings page to create ML jobs per environment {kibana-pull}70560[#70560]
+* Service maps anomaly detection integration by environment {kibana-pull}70932[#70932]
+* Anomaly detection setup link with alert if job doesn't exist {kibana-pull}71229[#71229]
+* Respects default time range defined in Kibana Advanced Settings {kibana-pull}71464[#71464]
+* Uses HDR for percentiles {kibana-pull}64758[#64758]
+Dashboard::
+* Adds Lens drilldowns {kibana-pull}65675[#65675]
+* Improves duplicate panel positioning {kibana-pull}67461[#67461]
+Discover::
+* Multi-line kql bar {kibana-pull}70140[#70140]
+Ingest Management::
+* Adds support for datastream to each template {kibana-pull}66367[#66367]
+* Improves server-side error handling {kibana-pull}67278[#67278]
+* Adds ability to copy an agent config {kibana-pull}68737[#68737]
+* Adds enroll agent action to config action menu {kibana-pull}68840[#68840]
+* Adds ability to sort to agent configs and package configs {kibana-pull}70676[#70676]
+Kibana UI::
+* Adds link for Cloud deployment settings {kibana-pull}66486[#66486]
```

Comment:
I wonder if we should remove this - seems like it would only be applicable on the Cloud and those users will see it anyway?